Connectors/Rhino/Speckle.Connectors.GrasshopperShared/HostApp/Helpers.cs

```diff
@@ -96,6 +96,37 @@ public static Transform MatrixToTransform(Matrix4x4 matrix, string units)
     return t;
   }
 
+  public static Matrix4x4 TransformToMatrix(Transform rhinoTransform, string? units)
+  {
+    var currentDoc = RhinoDoc.ActiveDoc; // POC: too much right now to interface around
```

Comment:
hmm a little worried about using this in a static context, will need to test units and if doc unit changes are picked up on any component that uses this

Reply:
very valid. will confirm in [acceptance testing](https://linear.app/speckle/issue/CNX-1944/acceptance-testing-beta-release)